builder/llpcBuilder.h

```diff
@@ -274,6 +274,14 @@ class Builder : public IRBuilder<>
         Value*        pCoord,             // [in] Input coordinate <3 x float>
         const Twine&  instName = "") = 0; // [in] Name to give instruction(s)
 
+    // Create scalar or vector FP truncate operation with the given rounding mode.
+    // Currently the rounding mode is only implemented for float/double -> half conversion.
+    virtual Value* CreateFPTruncWithRounding(
```

Comment:
CreateFPTruncWithRounding -> CreateFpTruncWithRounding

Reply:
This would then be inconsistent with IRBuilder<>::CreateFPTrunc. Are you sure you want this inconsistency?